INSTALL

```diff
@@ -1,54 +1,53 @@
-qBittorrent - A BitTorrent client in C++ / Qt4
+qBittorrent - A BitTorrent client in C++ / Qt
 ------------------------------------------
 
-1) Compile and install qBittorrent with Qt4 Graphical Interface
+1) Compile and install qBittorrent with Qt graphical interface
 
   $ ./configure
   $ make && make install
   $ qbittorrent
 
-  will install and execute qBittorrent hopefully without any problems.
+  Will install and execute qBittorrent.
 
   Dependencies:
-    - Qt >= 4.6.0 (libqtgui, libqtcore, libqtnetwork, libqtxml, libqtdbus/optional)
+    - Qt >= 5.5.1
+      * libqtgui, libqtcore, libqtnetwork, libqtxml, libqtdbus/optional
 
-    - pkg-config executable
+    - pkg-config
 
-    - libtorrent-rasterbar by Arvid Norberg (>= 1.0.6)
-        -> http://www.libtorrent.net
-        Be careful: another library (the one used by rTorrent) uses a similar name.
+    - libtorrent-rasterbar >= 1.0.6 (by Arvid Norberg)
+        * https://www.libtorrent.org/
+        * Be careful another library (the one used by rTorrent) uses a similar name
 
-    - libboost >= 1.35.x (libboost-system)
+    - Boost >= 1.35
+        * libboost-system
 
-    - python >= 2.3 (needed by search engine)
-        * Run time only dependency
+    - Python >= 2.7.9 / 3.3.0 (optional, runtime dependency)
+        * Needed by search engine
 
-    - geoip-database (optional)
-        * If qBittorrent cannot find this database, it will try to resolve countries using the Internet but it will be a lot slower.
-        * Run time only dependency
-
-2) Compile and install qBittorrent without Qt4 Graphical interface
+2) Compile and install qBittorrent without Qt graphical interface
 
   $ ./configure --disable-gui
   $ make && make install
-  $ qbittorrent
+  $ qbittorrent-nox
 
-  will install and execute qBittorrent hopefully without any problems.
+  Will install and execute qBittorrent.
 
   Dependencies:
-    - Qt >= 4.4.0 (libqt-devel, libqtcore, libqtnetwork)
-
-    - pkg-config executable
+    - Qt >= 5.5.1
+        * libqt-devel, libqtcore, libqtnetwork
```

Comment:
libqtxml

Reply:
OK.
Another question is libqt-devel is required here but not in GUI version? Should it be removed/replaced?

However before I push changes, I wonder is it really helpful to write all the lib names? Because different distro has different names for Qt packages, for example, an ArchLinux user will have a hard time figuring out what package should he install to satisfy libqtgui (by just reading INSTALL file).
I propose we should use Qt submodule names instead: https://download.qt.io/archive/qt/5.11/5.11.1/submodules/ , such as: `qtbase, qtsvg, qttools, qtwinextras (on Windows)`.

Reply:
I would vote for mentioning only Qt as a general dependency. Packages might change names in next major version of Qt like they did between qt4 and qt5.
IMO configure/cmake/qmake errors would provide insights to the builder on what's missing

Reply:
OK, done.